release/pkg/versions.go

```diff
@@ -54,16 +41,21 @@ func (v *versioner) buildMetadata() (string, error) {
 	return out, nil
 }
 
-func (v *versioner) patchVersion() (string, error) {
-	cmd := exec.Command("git", "-C", v.pathToProject, "describe", "--tag")
-	out, err := execCommand(cmd)
-	if err != nil {
-		return "", errors.Wrapf(err, "failed executing git describe to get version in [%s]", v.pathToProject)
+func (v *versioner) patchVersion(r *ReleaseConfig) (string, error) {
+	var gitTag string
+	if v.pathToProject == r.CliRepoSource {
```

Comment:
could we have different "versioners" and avoid the if-else?
that was the idea of injecting `projectVersioner`, that we could have different logics for different projects without hardcoding them in a switch

Reply:
Ahh good call. I'll make that change.

Reply:
Done